doc/source/examples/incompressible-flow/3d-flow-over-periodic-hills/3d-flow-over-periodic-hills.rst

```diff
@@ -0,0 +1,280 @@
+
+Due to the complexity of this example we recommend that you run this example using a cluster or supercomputer if available. For this it is necessary to add the ``mpirun -np X`` command at the beginning of the line. The number of processes ``X`` must be adjusted according to the machine. If you want to run this in a normal desktop we recommend that you set the parameter ``time end`` to ``5.0``; this allows you to observe the initial behavior of the simulation.
+
+Results
+-------
+To summarize, a coarse mesh of 250K cells was simulated, using a time step of 0.1 and taking average quantities between 207s and 1000s. The results are compared against established test data from both experiments and another CFD simulation. The experimental data corresponds to the data obtained from Rapp [2] and the computational data is extracted from the results of the LESOCC CFD code by Breuer et al. [3]. 
+
+The following image shows the average velocity profiles in the x-direction:
+
+.. image:: images/average_velocity.png
+    :alt: average velocity
+    :align: center
+    :name: average_velocity
+
+The values obtained for the reynolds normal stress in the x-direction:
+
+.. image:: images/reynolds_normal_stress.png
+    :alt: reynolds normal stress
+    :align: center
+    :name: reynolds_normal_stress
+
+and the Reynolds shear stress:
+
+.. image:: images/reynolds_shear_stress.png
+    :alt: reynolds shear stress
+    :align: center
+    :name: reynolds_shear_stress
+
+It can be seen that there is a very good agreement of the Lethe average velocity with the values of both benchmarks. Especially at the lower wall and in the bulk of the flow. The Reynolds stresses are more sensitive than the average velocity, as we can see a bigger difference near to the separation of the flow and the reattachment zone. For the periodic hills case, the reattachment point is a good indicator of the accuracy of the simulation at the near wall region. The reattachment point reported experimentally is 4.83 and for the LESOCC code is 5.09. The value obtained with this simulation is 4.73. This is shorter than the ones reported in the literature and a possible reason for the under-prediction is the coarse mesh that is being used. This value could be further improved by using a finer mesh. In general, this example shows that Lethe can be used to simulate complex flow problems with good accuracy and coarse meshes.
+
+Possibilities for extension
+----------------------------
+
+- **Parameter tuning**: It is possible to play with different parameters of the simulation such as time average, time step and number of cells of the mesh and see the effects on the results.
+
+- **High-order elements**: It would be interesting to observe the effect of high-order elements in the simulation of the periodic hills flow. For example, Q2-Q2 elements. The only part of the parameter file that would need to change would be the ``FEM`` section.
```

Comment:
Other examples added this remark within the FEM section, maybe we could have them at both places (only a suggestion).